styles.css

```diff
@@ -12,7 +12,7 @@ header {
   top: 0;
   left: 0;
   right: 0;
-  background-color: white !important;
+  background-color: #fff !important;
 }
```

Comment:
- [ ] Hi @Marlyn-Mayienga, I claimed your review, Pls can you kindly run `npx stylelint --fix "**/*.{css,scss}"` to fix all stylelint errors.